bom/pom.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <properties>
+        <aws-java-sdk-core.version>1.12.169</aws-java-sdk-core.version>
```

Comment:
> Maybe let's add it to https://github.com/quarkiverse/quarkus-amazon-services/blob/main/pom.xml#L40 instead?
> 
> Probably a good idea to make it clear it's v1 in the property name and that it is only used by Localstack and shouldn't be used for other things?
> 
> And now that I have reviewed the whole thing, I wonder if we should let localstack pick the version? Or we actually need to define the dependency ourselves?

I'll try to find out why it is required and if it can be changed at testcontainer repo but as @stuartwdouglas said, by default, if not in the path, the localstack class do not load at runtime.

Reply:
https://github.com/testcontainers/testcontainers-java/issues/4279

Reply:
OK, thanks for the explanations. Could you move this property with the other one then? And make sure its name clearly indicates it's the v1 version and it should not be used and only there for Testcontainers localstack. Probably a good idea to add a pointer to the issue you pointed out too.